app/models/slide.rb

```diff
@@ -92,7 +94,8 @@ def css_classes
     classes.join(' ')
   end
 
-  def take_screenshot
+  def take_screenshot(opt = false)
+    @skip_file_validation = opt
```

Comment:
I really didn't want the skip_file_validation variable to be persisted at all. This was my shot at attempting to make that work by using ruby attribute accessors and ignoring the db completely. You're totally right though, the `take_screenshot` method is now doing more than it should. I'll go back and fix this up (includes the previous line note you made as well)